content/en/docs/concepts/architecture/nodes.md

```diff
@@ -160,6 +160,16 @@ to be unreachable. (The default timeouts are 40s to start reporting
 ConditionUnknown and 5m after that to start evicting pods.) The node controller
 checks the state of each node every `--node-monitor-period` seconds.
 
+In versions of Kubernetes prior to 1.13, NodeStatus is the heartbeat from the
+node. Starting from Kubernetes 1.13, node lease feature is introduced, so that
+both NodeStatus and node lease are treated as heartbeats. When node lease
+feature is enabled, node leases are renewed frequently while NodeStatus is
```

Comment:
Is there a feature flag to enable node leases? If so, please add information on how that flag.

Reply:
Done